README.md

```diff
@@ -200,6 +200,10 @@ For information about the governance of the Node.js project, see
 * [Trott](https://github.com/Trott) -
 **Rich Trott** &lt;rtrott@gmail.com&gt; (he/him)
 
+<details>
+
+<summary>TSC emeriti</summary>
+
 ### TSC emeriti
```

Comment:
I'm tempted to remove this now that it's in the `summary` tag, but doing so would probably break `node-core-utils` which parses the README file. In fact, the change as it stands here might already do that. (I don't think so, but worth checking.)